app/src/features/mocks/container.scss

```diff
@@ -0,0 +1,23 @@
+.mocks-container {
```

Comment:
No need to define this css. This should get autodefined when you wrap SecondarySidebarLayout